templates/osbuild-worker.toml.j2: add channel for logs

schuellerf opened the pull request:
add the deployment channel to the logs.

Adding to splunk to work on all logs, need some alignment with the splunk maintainers to get it indexed properly (maybe?) and/or might impact performance here and there.

schuellerf committed:
templates/osbuild-worker.toml.j2: add channel for logs

schuellerf committed:
github/workflows/tests.yml: update dependencies for F40

thozza commented:
Thanks for the PR. You may need to drop c8s from the testing matrix, because it is EOL.

schuellerf commented:
And the tests seem to be outdated since fedora 40 anyways - right?

schuellerf committed:
github/workflows/tests.yml: drop centos stream 8 - EOL

schuellerf commented:
@thozza you mean this, right?
https://github.com/osbuild/ansible-osbuild-worker/pull/6/commits/5958680dc656ffe58eebe08006778dd8af7d366a

thozza commented:
> @thozza you mean this, right? [5958680](https://github.com/osbuild/ansible-osbuild-worker/commit/5958680dc656ffe58eebe08006778dd8af7d366a)

Yes

thozza commented:
> And the tests seem to be outdated since fedora 40 anyways - right?

Tests should be distro agnostic, but the setup part may need some :heart: to make them run again... :roll_eyes:

schuellerf committed:
molecule: replace Dockerfile.j2 and molecule.yml with symbolic link

Replace Dockerfile.j2 and molecule.yml with symbolic link
where possible to avoid code duplication
Followup of previous "Show the version of osbuild in tests" to show them
in all tests.

schuellerf committed:
tasks/deploy_worker.yml: fix installation of osbuild-composer-worker

due to an upstream bug "installed" does not work.
We need to switch to "present" until this is released everywhere
but we can also leave "present" as this causes the same behavior anyway
see: https://github.com/ansible/ansible/issues/83960

schuellerf committed:
molecule/default/verify.yml: Show the version of osbuild in tests

To help track down problems in github tests, we need to show
the version of osbuild installed.

schuellerf commented:
In the tests I introduced the molecule verify task "Show osbuild-composer-worker package details"
this needs to result in v120 or greater for the tests to work… (I hope)

schuellerf committed:
molecule/default-extended/verify: fix problem finding var

for some reason, the variable is not found.
Let's try if ansible finds it when it's explicitly set with
set_fact.
